mathesar_ui/src/systems/table-view/constraints/ConstraintCollapseHeader.svelte

```diff
@@ -53,10 +35,9 @@
   .column-names {
     display: flex;
     flex-direction: row;
-
-    > :global(* + *) {
-      margin-left: 0.25rem;
-    }
+    flex-wrap: wrap;
+    justify-content: flex-start;
+    gap: 0.25rem;
```

Comment:
We are trying to use this property because of less support in the Safari browser. You can use the owl selector in conjunction with `:global`. Something like: 

```css
> :global(* + *) {
    margin-left: 0.25rem;
}
```

Reply:
updated and avoided the use of gap property here